Gems/Stars/Code/CMakeLists.txt

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+ly_add_target(
```

Comment:
Might I suggest to also add HEADERONLY type target of Starts.API that other gems could use.

The Stars.Static library should not be used outside of the Stars gem targets